search/search_test.go

```diff
@@ -398,6 +400,34 @@ func TestTarjanSCC(t *testing.T) {
 	}
 }
 
+// batageljZaversnikGraph is the example graph from
```

Comment:
I'm happy to do that in a later change (I think that this has merit for a number of reasons - including as a teaching resource), but I'd like to just get the place clean first.

I'll file an issue.